src/sentry/utils/sdk.py

```diff
@@ -453,14 +454,16 @@ def flush(
         should_summarize_metric=minimetrics.should_summarize_metric,
     )
 
+    cache_spans = "CANARY" == os.environ.get("SENTRY_ENVIRONMENT", "").capitalize()
```

Comment:
Why the capitalization?

Reply:
I just thought it was more robust, and wasn't 100% on the casing of the environment names. If its definetely all lowercase i can remove the `capitalize`

Reply:
I mean, we're looking at a specific value here, the one we set for canary. If you're not sure if it's capitalized or not, check the ops repo to see what value we put here otherwise, I think it's unneeded.

Reply:
I'll dig around and see what i can find